keyboards/wsk/alpha9/config.h

```diff
@@ -0,0 +1,56 @@
+/* Copyright 2020 Worldspawn <mcmancuso@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x5753  // "WS" - Worldspawn00
+#define PRODUCT_ID      0x692A
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Worldspawn00
+#define PRODUCT         Alpha9
+
+/* key matrix size */
+#define MATRIX_ROWS 3
+#define MATRIX_COLS 13
+
+/* key matrix pins */
+#define MATRIX_ROW_PINS { D4, B4, B5 }
+#define MATRIX_COL_PINS { D7, E6, C6, B6, B2, B3, B1, F7, F6, F5, D1, D0, D2 }
+#define UNUSED_PINS
+
+/* COL2ROW or ROW2COL */
+#define DIODE_DIRECTION COL2ROW
+
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCE 5
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+#define RGB_DI_PIN F4
```

Comment:
Eg;

https://docs.qmk.fm/#/feature_led_indicators?id=configuration-options

```suggestion
#define LED_CAPS_LOCK_PIN D3
#define LED_PIN_ON_STATE 0

#define RGB_DI_PIN F4
```

Reply:
I have no problem with this at all, in fact it's way cleaner.  I think the only unfortunate thing is some of the other Worldspawn boards passed muster using the functions in board.c